frontend/app/helpers/column-stack.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
```

Comment:
Is there a case for putting package-lock.json in the gitignore?
I didn't manually change it. After you added eslint dependencies, I got missing dep errors, so I ran 'npm install' , which is probably what changed it